### DISCUSSION TITLE
Getting the interface to work

### DISCUSSION THREAD

**Discussion by WolfoTheWolf** [TGRHavoc/live_map#61] [Q&A] [answered]
Hey there im new to fivem (making servers) and I dont think I did this right.. 
![Screenshot (5)](https://user-images.githubusercontent.com/56661068/93828517-84d35800-fc39-11ea-9eba-76f4fd13ce33.png)
![Screenshot (6)](https://user-images.githubusercontent.com/56661068/93828518-856bee80-fc39-11ea-8f34-de2ef380bfc6.png)

**Comment by TGRHavoc** [comment]
Should be right? What's your question?

The "socketPort" should be the same in the config file as it is in your "server.cfg" file. And you need to make sure it's allowed through your firewall.

**Comment by WolfoTheWolf** [comment]
So tbh im not even 100% if im doing this right, first I went to added the "live_map" then went to my CFG and put "set socket_port 2dcrp
set livemap_debug "warn" # "[all]" 'trace', 'debug', 'info', 'warn', 'error', 'fatal', 'off'
set blip_file "server/blips.json"
set livemap_access_control "*"

start live_map"

--------------------
I then installed the live map interface (please keep in mind im currently trying to localhost) I edited the config.json as provided i the picture below. Started my server and put in my ip and my port, along with the socket port I created. I try to localhost to it localhost:66.221.101.164 (not sure if I done that right) And nothing comes up.

----------------IMAGES-------------------
![Screenshot (95)](https://user-images.githubusercontent.com/56661068/98435778-cf9f1700-20a3-11eb-9208-f34ca020fe24.png)
![Screenshot (96)](https://user-images.githubusercontent.com/56661068/98435779-cf9f1700-20a3-11eb-9dbc-7635f662b0ef.png)
![Screenshot (97)](https://user-images.githubusercontent.com/56661068/98435780-d037ad80-20a3-11eb-91f6-baaa8ace5362.png)

**Comment by TGRHavoc** [comment]
> So tbh im not even 100% if im doing this right, first I went to added the "live_map" then went to my CFG and put "set socket_port 2dcrp

The socket port should be a number. By default it's `30121` (one higher than the port FiveM uses). So, `2drcp` wouldn't work.
 
> I try to localhost to it localhost:66.221.101.164 (not sure if I done that right) And nothing comes up.

I'm not sure what you mean by this. If the interface and FiveM server are both running on the same machine you're trying to open the interface on, then you will be able to change the IP to `127.0.0.1` but, you _will_ need to change it to your external IP if you want other's to be able to use the interface themselves.

**Comment by WolfoTheWolf** [comment]
I guess the simple way to put what I'm asking is how to I connect to the interface. (I went back threw and fixed the 2dcrp and replaced it with 30121, and I'm trying to connect on the same machine. Sorry if I'm really stupid at this I'm not very familiar with ports, etcetera.

**Comment by TGRHavoc** [comment, marked as answer]
> I guess the simple way to put what I'm asking is how to I connect to the interface. (I went back threw and fixed the 2dcrp and replaced it with 30121, and I'm trying to connect on the same machine. Sorry if I'm really stupid at this I'm not very familiar with ports, etcetera.

You place the interface in a folder that your webserver can serve. By default I think Apache looks in `c:/Apache/Apache/htdocs` so, you could place it in there or, in it's own folder. Then, you modify the config.json file... It seems you've already done this so, you should be good to just go to the website and see what happens from there.

**Comment by WolfoTheWolf** [comment]
I just want to say this, thank you so much for helping me out and walking threw it with me. It really does mean alot and make sure you keep up the amazing work!

**Comment by WolfoTheWolf** [comment]
oh and it worked.
